src/pipdeptree/__init__.py

```diff
@@ -605,19 +605,29 @@ def mermaid_id(key: str) -> str:
     nodes: set[str] = set()
     edges: set[str] = set()
 
-    for pkg, deps in tree.items():
-        pkg_label = f"{pkg.project_name}\\n{pkg.version}"
-        pkg_key = mermaid_id(pkg.key)
-        nodes.add(f'{pkg_key}["{pkg_label}"]')
-        for dep in deps:
-            edge_label = dep.version_spec or "any"
-            dep_key = mermaid_id(dep.key)
-            if dep.is_missing:
-                dep_label = f"{dep.project_name}\\n(missing)"
-                nodes.add(f'{dep_key}["{dep_label}"]:::missing')
-                edges.add(f"{pkg_key} -.-> {dep_key}")
-            else:
-                edges.add(f'{pkg_key} -- "{edge_label}" --> {dep_key}')
+    if isinstance(tree, ReversedPackageDAG):
+        for pkg, rdeps in tree.items():
```

Comment:
please make variables valid english words seprated with snake case, so not rdep or rdeps